static/js/compose.js

```diff
@@ -276,6 +279,13 @@ exports.send_message = function send_message(request) {
 
     function success(data) {
         exports.send_message_success(local_id, data.id, locally_echoed);
+        // If 'Sending...' notification is active
+        if (not_in_view_notification) {
+            notifications.notify_above_composebox("Your message was sent!", "compose_notification_narrow_by_topic", local_id, "");
```

Comment:
I think we want this notification content to mention that they can see their message by scrolling down.